ursh/setup.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name='indico-plugin-ursh',
+    version='1.0',
```

Comment:
`3.0-dev` since it requires indico 2.2, and is currently under development. Basically 1.x plugins are for indico 2.0, 2.x plugins require 2.1, 3.x plugins require 2.2, ...